core/services/chainlink/application.go

```diff
@@ -923,6 +924,20 @@ func newCREServices(
 
 				engineRegistry := syncer.NewEngineRegistry()
 
+				globalLogger.Debugw("Creating WorkflowRegistrySyncer")
```

Comment:
Moved this up so that version can be passed to both the event handler and the WR Syncer